extensions/wikia/AdEngine/js/video/player/porvata/porvataPlayerFactory.js

```diff
@@ -36,6 +36,21 @@ define('ext.wikia.adEngine.video.player.porvata.porvataPlayerFactory', [
 			addEventListener: function (eventName, callback) {
 				ima.addEventListener(eventName, callback);
 			},
+			computeVastMediaAspectRatio: function () {
+				var aspectRatio = width / height,
+					currentAd;
+
+				if (ima.getAdsManager()) {
+					currentAd = ima.getAdsManager().getCurrentAd();
+					aspectRatio = currentAd.getVastMediaWidth() / currentAd.getVastMediaHeight();
+
+					if (!aspectRatio) {
```

Comment:
when will be the case when aspectRatio here will be undefined or 0? From what I see in docs () both `getVastMediaWidth` and `getVastMediaHeight` return the 0 in the same case: `The VAST media height/width of the selected creative or 0 if none is selected.` - so we'll have in line 45 case when we want to divide by 0. In JS division by 0 gives `Infinity` which used in conditional statement returns `true` ! In that case check `if (!aspectRatio)` doesn't satisfy us!